src/data/faq.json

```
@@ -500,7 +500,7 @@
                             "<b> Questions and answers about the Event QR code:</b>",
-                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can event QR codes from other apps be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Will there be a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan a Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",
+                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Will there be a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan a Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",

```

```suggestion
                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Will there be a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",
```
Grammar correction. Please also correct Line 589 as well!

---

Changes 
"What happens if I scan a Event QR code but do not have the CWA installed on my smartphone?" to
"What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?"

---

Thank you! Done in https://github.com/corona-warn-app/cwa-website/pull/1482/commits/2e115eec7a5f38de455441911ceee009eddc930e.